workers/unity/Assets/Playground/Scripts/MonoBehaviours/SpawnCubeCommandReceiver.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using System.Collections.Generic;
+using Generated.Improbable;
+using Generated.Improbable.Transform;
+using Generated.Playground;
+using Improbable.Gdk.Core;
+using Improbable.Gdk.Core.GameObjectRepresentation;
+using Improbable.Worker;
+using Improbable.Worker.Core;
+using Playground;
+using UnityEngine;
+using Quaternion = Generated.Improbable.Transform.Quaternion;
+using Transform = Generated.Improbable.Transform.Transform;
+
+#pragma warning disable 649
+
+public class SpawnCubeCommandReceiver : MonoBehaviour
+{
+    [Require] private Transform.Requirables.Reader transformReader;
+    [Require] private CubeSpawner.Requirables.CommandRequestHandler commandRequestHandler;
+    [Require] private CubeSpawner.Requirables.Writer writer;
+    [Require] private WorldCommands.WorldCommandRequestSender worldCommandRequestSender;
+    [Require] private WorldCommands.WorldCommandResponseHandler worldCommandResponseHandler;
+
+    private ILogDispatcher logDispatcher;
+    private WorldCommandHelper worldCommandHelper;
+
+    public void OnEnable()
+    {
+        commandRequestHandler.OnSpawnCubeRequest += OnSpawnCubeRequest;
+
+        worldCommandHelper = new WorldCommandHelper(
+            worldCommandRequestSender,
+            worldCommandResponseHandler
+        );
+
+        logDispatcher = GetComponent<SpatialOSComponent>().LogDispatcher;
+    }
+
+    public void OnDisable()
+    {
+        worldCommandHelper?.Dispose();
+        worldCommandHelper = null;
+
+        commandRequestHandler.OnSpawnCubeRequest -= OnSpawnCubeRequest;
```

Comment:
Is this one needed if the others aren't.

Reply:
Do you mean the deregistry of command request handling? Yes I will revert these in this PR